torchtitan/distributed/utils.py

```diff
@@ -205,11 +214,11 @@ def create_context_parallel_ctx(
     try:
         from torch.distributed.tensor.experimental import context_parallel
         from torch.distributed.tensor.experimental._attention import set_rotate_method
-    except ImportError:
-        print(
+    except ImportError as e:
+        raise ValueError(
             f"PyTorch version {torch.__version__} does not include the experimental "
             "Context Parallel API. Please update to a newer version."
-        )
+        ) from e
 
```

Comment:
TODO for me: `set_rotate_method` has been in the main PyTorch for a while. We should be able to remove this try/except.